### ISSUE TITLE
Drag and drop does not work as expected

### ISSUE THREAD

**Issue by delyan-kirov** [Anut-py/h-raylib#48]
Trying to drag and drop files fails. Particularly using the function `loadDroppedFiles` will seg fault while trying to free an invalid pointer to the file path:

```
free(): invalid pointer
Aborted (core dumped)
```

Or it will try to double free:

```
free(): double free detected in tcache 2
Aborted (core dumped)
```

It does that on the next frame or the frame after. I'm not sure if this is user error, because I don't know what I should or should not free manually and how. This is a basic version of what I tried:

```
mainloop :: AppState -> IO AppState
mainloop appState = do
  fileIsDropped <- isFileDropped
  when fileIsDropped $
    print =<< loadDroppedFiles
  endDrawing
  return appState''

main = whileWindowOpen_ mainloop =<< initApp

```

**Comment by delyan-kirov** [comment]
While working on my little applet, I found that using the foreign C functions will not crash the app and will give the correct pointer and the correct file path. 

```
  fileIsDropped <- isFileDropped
  when fileIsDropped $ do
    filePtr <- c'loadDroppedFiles
    print filePtr
    fileData <- peek filePtr
    print fileData
    c'unloadDroppedFiles filePtr
```

**Comment by Anut-py** [comment]
Fixed in 5.1.2.0
